bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -10,8 +10,8 @@ path = "fake_lib.rs"
 env_logger = "0.9"
 anyhow = "1.0"
 once_cell = "1.3"
```

Comment:
`once_cell` should be update to `1.12` to match upstream.

(from https://github.com/bytecodealliance/wasmtime/blob/v9.0.2/Cargo.toml)